setup.py

```diff
@@ -7,7 +7,7 @@
 with open(os.path.join(os.getcwd(), "requirements.txt")) as f:
     REQUIREMENTS = f.read().splitlines()
 
-VERSION = "2.0.1"
+VERSION = "2.0.2"
```

Comment:
Wondering what kind of version change this should be. Feels like at least a minor change (new attributes for clan capital). Here's a helpful read for things to think about with versioning, not sure if you've seen it: https://semver.org/

Reply:
I definitely think it's a minor update as well.